exonum-light-client/src/main/java/com/exonum/client/response/ConsensusStatus.java

```diff
@@ -15,14 +15,10 @@
  *
  */
 
-package com.exonum.client;
+package com.exonum.client.response;
 
-import lombok.Value;
-
-/**
- * Json object wrapper for health check response.
- */
-@Value
-class HealthCheckResponse {
-  boolean connectivity;
+public enum ConsensusStatus {
+  ACTIVE,
```

Comment:
Do you happen to know the difference between these two? As it is public, I'd document it.